plugins/oauth/server/providers/github.py

```diff
@@ -98,18 +97,13 @@ def getUser(self, token):
         email = emails[0]
 
         # Get user's OAuth2 ID, login, and name
-        resp = self._getJson(method='GET', url=self._API_USER_URL,
-                             headers=headers)
+        resp = self._getJson(method='GET', url=self._API_USER_URL, headers=headers)
         oauthId = resp.get('id')
         if not oauthId:
             raise RestException('GitHub did not return a user ID.', code=502)
 
-        login = resp.get('login', None)
+        login = resp.get('login', '')
+        names = (resp.get('name') or login).split()
+        firstName, lastName = names[0], names[-1]
```

Comment:
That's gonna create 'login login', wasn't it better with the previous default for `lastName` to `''`?